boards/arm/colibri_imx7d_m4/doc/colibri_imx7d_m4.rst

```diff
@@ -0,0 +1,339 @@
++---------------+-----------------+---------------------------+
+| OCRAM_S       | 0x00180000      | 32KB                      |
++---------------+-----------------+---------------------------+
+| OCRAM         | 0x00900000      | 128KB                     |
++---------------+-----------------+---------------------------+
+| DDR           | 0x80000000      | 2048MB                    |
++---------------+-----------------+---------------------------+
+
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| Region     | Cortex-A7             | Cortex-M4 (System Bus) | Cortex-M4 (Code Bus)  | Size                 |
++============+=======================+========================+=======================+======================+
+| DDR        | 0x80000000-0xFFFFFFFF | 0x80000000-0xDFFFFFFF  | 0x10000000-0x1FFEFFFF | 2048MB (less for M4) |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM_PXP  | 0x00940000-0x00947FFF | 0x20240000-0x20247FFF  | 0x00940000-0x00947FFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM_EPDC | 0x00920000-0x0093FFFF | 0x20220000-0x2023FFFF  | 0x00920000-0x0093FFFF | 128KB                |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM      | 0x00900000-0x0091FFFF | 0x20200000-0x2021FFFF  | 0x00900000-0x0091FFFF | 128KB                |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCMU       | 0x00800000-0x00807FFF | 0x20000000-0x20007FFF  |                       | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCML       | 0x007F8000-0x007FFFFF |                        | 0x1FFF8000-0x1FFFFFFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM_S    | 0x00180000-0x00187FFF | 0x20180000-0x20187FFF  | 0x00000000-0x00007FFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| QSPI Flash |                       |                        | 0x08000000-0x0BFFFFFF | 64MB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+
+
+References
+==========
+
+- `i.MX 7 Dual Reference Manual`_ from page 190 (section 2.1.2 and 2.1.3)
+- `Toradex Wiki`_
+
+
+At compilation time you have to choose which RAM will be used. This configuration is
+done in the file ``boards/arm/colibri_imx7d_m4/colibri_imx7d_m4.dts`` with "zephyr,flash" (when CONFIG_XIP=y) and "zephyr,sram"
+properties. The available configurations are:
+
+"zephyr,flash"
```

Comment:
Since these are lines in the dts file, use a code block:

    .. code-block:: none

      "zephyr, flash"
      - &ddr_code